Source/EventFlow.AspNetCore/Configuration/EventFlowJsonOptionsMvcConfiguration.cs

```diff
@@ -27,7 +27,13 @@
 
 namespace EventFlow.AspNetCore.Configuration
 {
-    public class EventFlowJsonOptionsMvcConfiguration : IConfigureOptions<MvcJsonOptions>
+    public class EventFlowJsonOptionsMvcConfiguration : 
+#if NETSTANDARD2_0
+        IConfigureOptions<MvcJsonOptions>
+#endif
+#if NETCOREAPP3_0
+        IConfigureOptions<MvcNewtonsoftJsonOptions>
```

Comment:
I think that using ASP.NET Core 2.1 on .NET Core is possible. This scenario will fail, isn't it?